dogs/.gitignore

```diff
@@ -7,7 +7,6 @@
 /bazel-out
 
 # Node
-/node_modules
```

Comment:
why?

Reply:
i wait next call to talk about this "problem"